float64[] nUnit
float64[] goal

string motionType
---
bool success
